### PR TITLE
Accessibility (a11y) improvements to exercismo.io

### PR DESCRIPTION
**Pull request by Tetsuro**
PR for https://github.com/exercism/exercism.io/issues/3066 to improve accessibility across the site. 

## 1. Add landmarks

- Wrapped content area with `<main role="main">`
- Wrapped footer area with `<footer role="contentinfo">`
- Wrapped sidebar in `language.erb` with `<aside role="complementary">`

Before | After
-------|---------
landmarks1 | landmarks2

## 2. Remove screen reader noise

- Updated the `track_icon` helper to apply a blank `alt` attribute so that the icon's file name is not read by screen readers. 
- Applied a similar treatment for the icons in the footer.

Before | After
-------|---------
screen shot 2016-09-06 at 11 14 20 pm | screen shot 2016-09-06 at 11 14 53 pm
screen shot 2016-09-06 at 11 16 39 pm | screen shot 2016-09-06 at 11 17 18 pm

## 3. Add supplementary text for hidden text

I noticed that the label next to the icon in the Sponsors list disappears on smaller screens. As such I added a supplementary visually hidden `span` class so that the Sponsor name is read on all screen sizes.

Before | After
-------|---------
screen shot 2016-09-06 at 11 18 39 pm | screen shot 2016-09-06 at 11 18 50 pm

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@ postgres: &postgres
   pool: 5
   host: <%= ENV["DEV_DATABASE_HOST"] || "localhost" %>
   port: 5432
-  username: exercism
-  password: apples
+  username: tetchi-mini
+  password:
```
Comment (Insti) [inline config/database.yml]: These changes should probably not have been checked in.

Comment (Tetsuro) [inline config/database.yml]: yeah sorry, it was getting late and didn't realize I committed a bunch of stuff that I didn't mean to lol (including compiled CSS). Fixing later today.

**Comment by kytrinyx** [comment]
First of all: Yay!

Once you're ready for a review, would you mind rebasing onto master and also posting screenshots of the before/after for the changes you've made?

**Comment by Tetsuro** [comment]
Just as an update, this PR is pretty good to go except for this mess I've created in the compiled import.css 😅 https://github.com/Tetsuro/exercism.io/blob/ae5ba3874f5617e258cf3da3d4f5fae763f2eb4e/public/css/base/components/import.css

Will try and untangle that mess tomorrow.

**Comment by bernardoamc** [comment]
This looks great! 

Have you run compass with the production flag? `bundle exec compass compile --production`

**Comment by Tetsuro** [comment]
Hmm @bernardoamc do you know why I'd be getting `Error: invalid option: --production` when I try to run that command?

**Comment by bernardoamc** [comment]
Hey @Tetsuro ! Could you try `bundle exec compass compile -e production`?

**Comment by Insti** [comment]
@Tetsuro do you have more time to look at this? It would be good to get these changes merged in.

**Comment by Tetsuro** [comment]
@Insti oh man, I was **just** talking to @bernardoamc about this yesterday! I was having some trouble compiling the SCSS on my machine, we're hoping to take a look and get this merged today.

**Comment by Tetsuro** [comment]
Hey @bernardoamc, would you be able to merge this when you get a chance? I think it's good to go! :)

**Comment by kytrinyx** [comment]
Hi! I'm so sorry I dropped the ball on this. I'm going to resolve the conflicts and merge this now.

**Comment by kytrinyx** [comment]
Merged in a1476ecf1cde3443da098d6991b15e1ee1568709 :tada: :tada:

**Comment by Tetsuro** [comment]
Thank you @kytrinyx! and thanks @bernardoamc for your help with the SCSS stuff haha :)

**Comment by Insti** [comment]
And thank **YOU** @Tetsuro for all your work on this :heart:

**Pull request closed without merging**